Validation/RecoTau/python/dataTypes/ValidateTausOnZTT_cff.py

```diff
@@ -31,8 +31,23 @@
 zttModifier = ApplyFunctionToSequence(zttLabeler)
 proc.TauValNumeratorAndDenominatorZTT.visit(zttModifier)
 
+#Set discriminators
+discs_to_retain = ['ByDecayModeFinding', 'CombinedIsolationDBSumPtCorr3Hits', 'IsolationMVArun2v1DBnewDMwLT', 'IsolationMVArun2v1PWnewDMwLT']
```

Comment:
What about MVAIso for old DMs? Does it make sense to test efficiency of Electron/MuonRejaction on ZTT?